fenix/app/src/test/java/org/mozilla/fenix/onboarding/JunoOnboardingFragmentTest.kt

```diff
@@ -0,0 +1,87 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.onboarding
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.mozilla.fenix.R
+import org.mozilla.fenix.onboarding.view.OnboardingPageUiData
+
+class JunoOnboardingFragmentTest {
```

Comment:
I think this can be renamed to `OnboardingPageUiDataTest` as it's testing functions on List<OnboardingPageUiData> and not the fragment.